src/module/actor/actor-sheet.js

```diff
@@ -533,8 +533,20 @@ export class OseActorSheet extends ActorSheet {
       item.sheet.render(true);
     });
 
-    html.find(".item-delete").click((event) => {
-      this._removeItemFromActor(event);
+    html.find(".item-delete").click(async (event) => {
```

Comment:
This arrow function is async, but we're not awaiting anywhere. Should we be awaiting something?

Reply:
No, I missed that when I replaced the custom dialog with the Dialog.confirm - cleaned up in latest commit!